miniapps/gslib/findpts.cpp

```diff
@@ -41,6 +42,63 @@
 using namespace mfem;
 using namespace std;
 
+// Experimental - required for visualizing functions on p-refined spaces.
+GridFunction* ProlongToMaxOrder(const GridFunction *x, const int fieldtype)
```

Comment:
It would be nice to have this function in the `GridFunction` class, but it may need to be generalized for any type of `FiniteElementCollection`, so I am fine with leaving it here for now (users can copy it for their own needs).